new maps

ScottyXIII opened the pull request.

norgeous commented on src/objects/entities/Ben3.ts:
```diff
@@ -102,6 +102,8 @@ class Ben3 extends Entity {
 
   public headScale = HEAD_SCALE_MIN;
 
+  private isJumping = false;
```
`isJumping` feels more like a state variable that indicates if the Ben3 is jumping

thiis is more like a `hasUsedJumpSinceTouchingGround` maybe?

i think traditionally "Coyote Jumps" (or "Coyote Time") is done with a timer in addition to what you have here

ScottyXIII commented:
Added 2 new maps. and 3rd map being the OG map we had.
Added scenes for the map. (copy pasta of GameScene with change of json map data) - if we had more time this should be abstracted.
